lib/dl_connector_bundle_chs3/dl_connector_bundle_chs3_tests/db/base/core/base.py

```diff
@@ -193,3 +197,28 @@ def sample_file_data_source(self, sample_s3_file: str) -> BaseFileS3Connection.F
     @pytest.fixture(scope="function")
     def connection_creation_params(self, sample_file_data_source: BaseFileS3Connection.FileDataSource) -> dict:
         return dict(sources=[sample_file_data_source])
+
+    @pytest.fixture(scope="function")
+    def sample_file_data_source_2(
+        self,
+        sample_file_data_source: BaseFileS3Connection.FileDataSource,
+    ) -> BaseFileS3Connection.FileDataSource:
+        new_source = copy.deepcopy(sample_file_data_source)
+        new_source.id = str(uuid.uuid4())
+        new_source.file_id = str(uuid.uuid4())
+
+        return new_source
+
+    @pytest.fixture(scope="function")
+    def connection_creation_params_2(self, sample_file_data_source_2: BaseFileS3Connection.FileDataSource) -> dict:
+        return dict(sources=[sample_file_data_source_2])
+
+    @pytest.fixture(scope="function")
+    def saved_connection_2(self, sync_us_manager: SyncUSManager, connection_creation_params_2: dict) -> FILE_CONN_TV:
```

Comment:
if it's only used in a single test, maybe we can create a second connection in the test itself?